### ISSUE TITLE
Add crossorigin="use-credentials" in javascript module to allow credentials.

### ISSUE THREAD

**Issue by jmesa-sistel** [angular/angular-cli#15125]
# 🚀 Feature request

### Command (mark with an `x`)

```
- [ ] new
- [X] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Description
 A clear and concise description of the problem or missing capability...
Add `crossorigin="use-credentials"` in javascript module to allow credentials.
In safari credentials are not send by default in a javascript module, so if the access to your webapp is protected with credentials in the backend, the module will not be loaded

### Describe the solution you'd like
 If you have a solution in mind, please describe it.
Just add `crossorigin="use-credentials"` in javascript module.
I found my problem after reading [https://jakearchibald.com/2017/es-modules-in-browsers/](https://jakearchibald.com/2017/es-modules-in-browsers/) in **No credentials** section

### Describe alternatives you've considered
 Have you considered any alternative solutions or workarounds?

**Comment by alan-agius4** [comment]
This feature will available in 8.2
Kindly check https://github.com/angular/angular-cli/pull/14844

**Comment by alan-agius4** [comment]
Hi, just a small correct re the above, it is available in 8.1.
